### ISSUE TITLE
Body Tracking broken on new Nvidia GPUs

### ISSUE THREAD

**Issue by gradientLord** [microsoft/Azure-Kinect-Sensor-SDK#1761]
Just got a new 3080ti system for Kinect development, none of the body tracking samples will start.

Here are the behaviors I'm getting for each mode:

CUDA:
-it spits out this error, and then closes:
```
2022-04-02 16:42:41.9449232 [W:onnxruntime:K4ABT_ORT_Environment, cuda_execution_provider.cc:1885 onnxruntime::CUDAExecutionProvider::GetCapability] CUDA kernel not found in registries for Op type: Sign node name: Sign
2022-04-02 16:42:41.9484857 [W:onnxruntime:K4ABT_ORT_Environment, fallback_cpu_capability.h:140 onnxruntime::GetCpuPreferedNodes] Force fallback to CPU execution for node: Shape__40
```

CPU:
-no error, it just opens and closes

DirectML:
-no error, it just opens and closes

TensorRT:
-it spits out this error, and then closes:
```
2022-04-02 16:52:07.6850310 [E:onnxruntime:K4ABT_ORT_Environment, tensorrt_execution_provider.h:45 onnxruntime::TensorrtLogger::log] [2022-04-02 21:52:07   ERROR] INVALID_ARGUMENT: getPluginCreator could not find plugin Sign version 1
Unsupported ONNX data type: DOUBLE (11)
Unsupported ONNX data type: DOUBLE (11)
Unsupported ONNX data type: DOUBLE (11)
Unsupported ONNX data type: DOUBLE (11)
2022-04-02 16:52:22.5834758 [W:onnxruntime:K4ABT_ORT_Environment, cuda_execution_provider.cc:1885 onnxruntime::CUDAExecutionProvider::GetCapability] CUDA kernel not found in registries for Op type: Sign node name: Sign

```
---

I've tried upgrading and downgrading drivers (can't downgrade below Driver 511.65) It's important I get this up and running for work, are there any options to get this working with a new nvidia card?

**Comment by aiwhoo** [comment]
I'm noticing a similar issue on a new laptop/new install of SDKs.

**Comment by gradientLord** [comment]
@aiwhoo were you able to find a solution/workaround?

**Comment by aiwhoo** [comment]
Unfortunately, no. Previous git issues indicate the sdk just needs to be updated.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by vilmire** [comment]
not working in 3080ti  
cuda : 11.1
gpu driver : 472.xx, also recent version of driver didnt

**Comment by gradientLord** [comment]
It seems people are getting the same behaviors with older cards with the latest Nvidia drivers.

**Comment by fkavache** [comment]
Hi @gradientLord 
could you please take a look at this thread?

https://github.com/microsoft/Azure-Kinect-Sensor-SDK/issues/1763#issuecomment-1091234984

**Comment by aiwhoo** [comment]
@gradientLord @fkavache Kinect viewer works for me; however the sample code fails. Previous threads indicate an issue with the SDK.

**Comment by fkavache** [comment]
@aiwhoo

So the simple_3d_viewer from SDK worked for you before, but the samples did not?

**Comment by aiwhoo** [comment]
@fkavache That is correct. 

Particularly, it fails when trying to verify(k4abt_tracker_create(&sensorCalibration,tracker_config,&tracker, "Body tracker initialization failed") step.

Usage: k4abt_jump_analysis_sample PROCESSING_MODE[CUDA, DirectML ( default ), or TensorRT](optional) -model MODEL_FILEPATH(optional).

The given version [10] is not supported, only version 1 to 7 is supported in this build.

[2022-04-10 14:41:17.960] [error] [t=7776] [K4ABT] ..\src\TrackerHost\TrackerHost.cpp (184): Create(). onnxruntime.dll was loaded at C:\Windows\SYSTEM32\onnxruntime.dll. But required APIs could not be found in this onnxruntime.dll! Wrong version of the onnxruntime.dll was loaded. Please place the onnxruntime.dll that shipped with the SDK at C:\Users\hxspo\Downloads\Azure-Kinect-Samples-mas\Azure-Kinect-Samples-master\body-tracking-samples\jump_analysis_sample\build\bin\Release

[2022-04-10 14:41:17.960] [error] [t=7776] [K4ABT] ..\src\sdk\k4abt.cpp (39): tracker->Create(sensor_calibration, config) returned failure in k4abt_tracker_create()

**Comment by fkavache** [comment]
@aiwhoo

It’s a different kind of issue, it seems like it was unable to find **onnxruntime.dll** in your packages and searched it in SYSTEM32 where there was an older version.

Which version of BT SDK are you using?

**Comment by aiwhoo** [comment]
@fkavache the most recent--fresh download and install on a new computer.

**Comment by aiwhoo** [comment]
@fkavache Any other suggestions on how to fix this issue? I've tried with older and newer onnxruntime.dll. Nuget seems to be broken with the 1.10, so I've even tried manually inserting in packages/system32

**Comment by fkavache** [comment]
Hi @aiwhoo

Latest BT SDK v1.1.1 version comes with nuget package.

https://docs.microsoft.com/en-us/azure/kinect-dk/body-sdk-download

**Comment by gradientLord** [comment]
> Hi @gradientLord could you please take a look at this thread?
> 
> [#1763 (comment)](https://github.com/microsoft/Azure-Kinect-Sensor-SDK/issues/1763#issuecomment-1091234984)

the new body tracking viewer in here works for me, I wasn't able to get the Tensor RT cache working yet but everything else seems good!

**Comment by qm13** [comment]
Fixed in v1.1.2
